test/read_events.h

```diff
@@ -16,6 +19,27 @@
 // STL
 #include <iostream>
 
+void processRun(const podio::Frame& run) {
+  //===============================================================================
+  // get generator tool info from the run
+  auto toolInfos = edm4hep::utils::getGenToolInfos(run);
+  auto toolinfo = toolInfos[0];
+  if (toolinfo.name != "something")
+    throw std::runtime_error("toolinfo.name != 'something'");
+  if (toolinfo.version != "v1")
+    throw std::runtime_error("toolinfo.version != 'v1'");
+  if (toolinfo.description != "some tool")
+    throw std::runtime_error("toolinfo.description != 'some tool'");
+
+  //===============================================================================
+  // get generator weight names
+  auto weightNames = run.getParameter<std::vector<std::string>>(edm4hep::generatorWeightNames);
```

Comment:
```suggestion
  auto weightNames = run.getParameter<std::vector<std::string>>(edm4hep::generatorWeightNames).value();
```

Looks like I missed this usage of parameters in the last review.